docs/guide/limitations.md

```diff
@@ -26,57 +26,144 @@ Currently this covers `Subtree of Location` and `Section` Limitations.
 
     Core Policies with Limitations are defined in [`EzPublishCoreBundle/Resources/config/policies.yml`](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/settings/policies.yml).
 
-Each Module contains functions, and for each function, you have Limitations. The default values are shown below.
+Each function in one of the four modules (content, section, state and user) can be assigned different Limitations.
```

Comment:
What about Workflow? It is listed below and has `Workflow Transition` limitation.